src/plugins/newsfeed/server/index.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { PluginConfigDescriptor } from 'kibana/server';
+import { NewsfeedPlugin } from './plugin';
+import { configSchema, NewsfeedConfigType } from './config';
+
+export const config: PluginConfigDescriptor<NewsfeedConfigType> = {
+  schema: configSchema,
+  exposeToBrowser: {
+    service: true,
+    defaultLanguage: true,
```

Comment:
Is it used? i18n should always have a locale set.

Reply:
you are right... it looks like it's only used to fall back `i18n.getLocale()` which should always return a language... Removing it from `exposeToBrowser` but not removing it from the config.ts file since we'll need to do a proper deprecation follow up. Added a deprecation warning instead.